Registry versioning requirements (#423)
w3c/wot-binding-templates: registry-requirements.md

    - Reading the protocol specification
    - Implementing a non-commercial device/Thing
    - Implementing a non-commercial Consumer application/driver
    - Conditions for commercial use, e.g., building a commercial product with the binding
    - Making a statement about your product's supporting that binding
  - If the entry depends on another one, it MUST specify the exact version of the dependency upon which it depends at the time of submission.

>   - All the previous versions of the entry MUST be listed as links.

Author: ```suggestion
  - The previous version of the summary document MUST be listed as a link.
```

>     - Conditions for commercial use: E.g. building a commercial product with the binding

Reviewer: ```suggestion
    - Conditions for commercial use, e.g., building a commercial product with the binding
```
